src/WignerSymbols.jl

```diff
@@ -365,13 +448,83 @@ function compute6jseries(β₁, β₂, β₃, α₁, α₂, α₃, α₄)
     return Base.unsafe_rational(totalnum, totalden)
 end
 
+# compute the sum appearing in the 9j symbol
+function compute9jseries(a, b, c, d, e, f, g, h, j)
+    I1 = max(abs(h - d), abs(b - f), abs(a - j))
+    I2 = min(h + d, b + f, a + j)
+    krange = I1:I2
+ 
+    sum(krange) do k
```

Comment:
I would prefer an explicit `return` in front of this `sum`, or first assigning it to a variable and then returning that. It took me a few seconds to realise this was the return value.

Reply:
Can do! I was split on whether the implicit return value was bad style myself.